_tutorials/customize-geometries.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: tutorial
+title: "Customizing Geometries"
+author: Alfonso Martínez (alfmarcua)
+tags: import lua
+---
+
+This tutorial will guide you through the process of customizing geometries in
+Nominatim. By customizing geometries, a more refined shape can be achieved for
+geographical elements, resulting in more precise and tailored search outcomes
+for specific use cases.
```

Comment:
It would be good to emphasize here that the goal is to use custom geometries instead of the geometries for OSM and to keep the custom geometries during updates even when data changes come in from OSM.

Reply:
I have added a couple of lines, please have a look.